rosidl_typesupport_cpp/src/type_support_dispatch.hpp

```diff
@@ -42,15 +39,24 @@ get_typesupport_handle_function(
     return handle;
   }
 
-#ifdef ROSIDL_TYPESUPPORT_CPP_USE_POCO
   if (handle->typesupport_identifier == rosidl_typesupport_cpp::typesupport_identifier) {
     const type_support_map_t * map = \
       static_cast<const type_support_map_t *>(handle->data);
     for (size_t i = 0; i < map->size; ++i) {
       if (strcmp(map->typesupport_identifier[i], identifier) != 0) {
         continue;
       }
-      Poco::SharedLibrary * lib = nullptr;
+      rcutils_shared_library_t * lib = nullptr;
+      rcutils_allocator_t allocator = rcutils_get_default_allocator();
+      lib = static_cast<rcutils_shared_library_t *>(allocator.allocate(
```

Comment:
Same comments as for the `rosidl_typesupport_c` file.

Reply:
Fixed in both files